_data/projects/lucene.net.yml

```diff
@@ -0,0 +1,13 @@
+name: Apache Lucene.Net
+desc: A port of the Lucene fulltext search engine library, written in C# and targeted at .NET runtime users.
+site: http://lucenenet.apache.org/
+tags:
+- .NET
+- C#
+- full-text
+- fulltext
+- lucene
+- search
+upforgrabs:
+- name: get involved
+- link: https://issues.apache.org/jira/browse/LUCENENET/
```

Comment:
We do a terrible job of having anything in JIRA, we mostly stick to our mailing list. We agreed amongst ourselves to fill out JIRA, if you want to move this back to WIT for the moment, OK. I will try to respond in a few days with a better URL and a bunch of issues.